datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java

```diff
@@ -164,4 +183,18 @@ private FlowIdentifier notify(String selector, SdxEvent acceptable) {
         }
 
     }
+
+    private String getBackupLocation(String datalakeName) {
```

Comment:
environmentClientService.getByName requires the environment's name, not sure how did this work

Reply:
You are right. It has to be fixed. It worked for me because I normally have the same name for both the environment and data lake. :-(